chart/values.yaml

```
@@ -157,6 +157,9 @@ jaeger-operator:
 
 agents:
   core:
+    # -- Request timeout for core agents
+    # Default value is defined in .base.default_req_timeout
+    requestTimeout: "5s"
```

Since it's the same value, can leave it as empty so it takes up the default, unless it's set

```suggestion
    # -- Request timeout for core agents
    # Default value is defined in .base.default_req_timeout
    requestTimeout:
```